### PR TITLE
Allow single node fusion for nvfuser

### PR DESCRIPTION
**Pull request by jjsjann123**
Setting `PYTORCH_NVFUSER_ONE_OP_FUSION=1` will take all nodes nvFuser support, instead of waiting for fusion opportunity.

### CODE REVIEW

**Comment by jjsjann123** [comment]
Cherry-picked from local devel branch per request on opinfo-based test.

**Comment by malfet** [comment]
Congrats, you've hit 70k PR number anniversary!

**Comment by eellison** [comment]
deferring to @davidberard98... but we probably want a runtime configurable switch like with https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/tensorexpr_fuser.cpp#L25 and getters/setters https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/python/init.cpp#L803

**Comment by jjsjann123** [comment]
Yeah would be even better to expose this on python APIs. We didn't do that in the initial PR simply to minimize our changes on pytorch code base. Please let us know how we would prefer to approach this.

**Review by davidberard98 — COMMENTED** [review]
@jjsjann123 could you add the python bindings like @eellison suggested?

**Comment by jjsjann123** [comment]
Oops, imported too early :laughing: 

Updated per request.

**Comment by jjsjann123** [comment]
FYI, I smuggled another API `_jit_set_nvfuser_horizontal_mode`, which is also a debugging knob and needed to WAR some PW scheduling bug.

File: test/test_jit_cuda_fuser.py [hunk 1]
```diff
@@ -3099,6 +3117,40 @@ def t(x: torch.Tensor, y: torch.Tensor):
...
+
+        with nvfuser_horizontal_fusion(False):
+            def t(x, y, s):
+                o1 = x + s
+                o2 = y + s
+                return o1, o2
+
+            t_jit = torch.jit.script(t)
+            for i in range(5):
+              t_jit(x, y, s)
```
Comment (davidberard98) [inline test/test_jit_cuda_fuser.py:3149]: linter complaining about this, can you indent?

**Review by davidberard98 — APPROVED** [review]
looks good other than the linter issue

File: test/test_jit_cuda_fuser.py (hunk 1 above)
Comment (jjsjann123) [reply test/test_jit_cuda_fuser.py]: Oops, fixed that in our internal branch but forgot to push it here.

**Review by davidberard98 — APPROVED** [review]

**Pull request closed without merging**